src/test/scala/fr/polytechnique/cmap/cnam/etl/extractors/patients/PatientsSuite.scala

```diff
@@ -79,16 +86,22 @@ class PatientsSuite extends SharedContext {
       ("Patient_01", 1, 69, makeTS(2014, 9, 26))
     ).toDF("NUM_ENQ", "MCO_FASTC__COD_SEX", "MCO_FASTC__AGE_ANN", "EXE_SOI_DTD")
 
-    val sources = new Sources(dcir = Some(dcirDf), mco = Some(mcoDf), irBen = Some(irBenDf), mcoCe = Some(mcoceDf))
+    val sources = new Sources(
+      dcir = Some(dcirDf),
+      mco = Some(mcoDf),
+      irBen = Some(irBenDf),
+      mcoCe = Some(mcoceDf),
+      ssr = Some(ssrDf))
 
     // When
-    val result = new Patients(config).extract(sources).toDF
+    val result = new Patients(config).extract(sources).toDF.sort("patientID")
```

Comment:
Do you need the sort here?

Reply:
yes, if I omit that, I get different sorting of the table at each run...

Reply:
But is shouldn't affect the assert, or is it affecting it?

Reply:
it is affecting the assert...

Reply:
My fault, it is not affecting it... the sort is removed now